NXDOC-2526: Upgrade webui notes from lts2021 to lts2023 (#2080)
nuxeo/doc.nuxeo.com-content: src/nxdoc/web-ui/web-ui-upgrade-notes/web-ui-upgrade-notes-lts-2023.md

---
title: Upgrading Nuxeo Web UI from LTS 2021 to LTS 2023
description: Upgrade notes from Nuxeo Web UI LTS 2021 to LTS 2023
review:
    comment: ''
    date: '2023-03-28'
    status: ok
toc: true
labels:

> tree_item_index: 998

Reviewer: Can't check this one now (can't run a build of LTS 2023 documentation due to technical issues) but this probably needs to be changed. Was 550 for LTS 2021.

> hidden: true

Reviewer: ```suggestion
```
Should not be hidden otherwise people won't be able to reach the page.

---

{{! multiexcerpt name='upgrade-notes'}}

{{! multiexcerpt name='matching-notes'}}
{{#> callout type='info' heading='Release Notes'}}
This page mentions how to do a technical upgrade. Have a look at the [release notes]({{page page='web-ui-release-notes'}}) to see what's new.
{{/callout}}
{{! /multiexcerpt}}

## Upgrading Web UI from LTS 2021 to LTS 2023

> 

Reviewer: ```suggestion
The initial release of Web UI on LTS 2023 is meant to be a transparent upgrade. It uses the same codebase as LTS 2021 and requires no specific upgrade process.
```

#### Studio Projects

> Studio configuration is made to be compatible and does not need to be migrated. If you did add customization in your project, see below for further details. Otherwise you can consider your project safe to use.

Reviewer: ```suggestion
Studio configuration for Web UI is made to be compatible and does not need to be migrated. You simply need to update your dependencies in the [application definition](insert link here) screen to their LTS 2023 counterpart in order to generate a package that will be compatible with LTS 2023.
```

> #### HTML Imports

Reviewer: ```suggestion
```
No need to mention this here, this would only confuse customers

> Nuxeo added a compatibility layer to keep HTML imports working. There is no need to update HTML imports done in your code.

Reviewer: ```suggestion
```
No need to mention this here, this would only confuse customers
